drivers/windows/R/bootstrap.R

```diff
@@ -1,31 +1,68 @@
-bootstrap_update <- function(development = NULL, root = NULL) {
+bootstrap_update <- function(development = NULL, root = NULL,
+                             platform = "windows") {
   path_script <- "hipercow/bootstrap-windows.R"
   path_root <- hipercow:::hipercow_root(root)$path$root
   path_script_abs <- file.path(path_root, path_script)
   dir.create(dirname(path_script_abs), FALSE, TRUE)
   bootstrap <- read_template("bootstrap.R")
+  prefix <- if (platform == "windows") "I:" else "/wpia-hn/Hipercow"
+  suffix <- if (platform == "windows") "" else "-linux"
```

Comment:
group as suggested before. But I do think that we possibly should just move immediately to to `-windows` too at which point the suffix calculation is very easy

Reply:
Refactored